tests/functional/wsp/test_wsp.py

```diff
@@ -103,6 +82,20 @@ def clean_dir(path='/tmp/WSP/'):
     shutil.rmtree(path, ignore_errors=True)
 
 
+@pytest.mark.parametrize(
+    'expected_results',
+    [
+        expected_json_results_from_file(
+            'functional',
```

Comment:
Prepend to this one the parameter name, so it's clear that is not the same as the rest.